tensorboard/plugins/text_v2/text_demo_v2.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Talked offline. We would like everything to be runnable by bazel. Please do one of two things:
1. remove `text_demo_v2.py` and just combine this into `text_demo.py` and test via `bazel run tensorboard/plugins/text:text_demo`
2. create a bazel target

Reply:
moved to original demo file